docs/developer-guide/running-basemaps-locally.md

```diff
@@ -0,0 +1,89 @@
+# Running Basemaps Locally
+
+This guide walks you through setting up and running a local instance of the **Basemaps** system on your machine. This guide shows you how to:
+
+- Build the packages in the **basemaps** repository
+- Run the **basemaps/server** package on your machine using either:
+    - [LINZ's imagery](#using-linzs-imagery)
+    - [your own imagery](#using-your-own-imagery)
+
+
+If you're planning to contribute to the **Basemaps** repository, make sure to review the [**CONTRIBUTING**][contributing] document.
+
+## Prerequisites
```

Comment:
Running basemaps server will require @bassemaps/landing page bundle. Which can just bundled from the repository or set `BASEMAPS_STATIC_ASSETS` to point to `s3://edge-staticbucket92253581-b8ekhlq4rs9d/` which contains landing page index.js.
https://github.com/linz/basemaps/blob/4bc33ff1a487f5894f4ca0c4607f146cf4e01096/packages/server/src/server.ts#L47

Reply:
basemaps server only needs the landing page if you want the `/` page to have LINZ branding, the tile server still works fine without it. so if you want to just use something like QGIS it would just "work"